Description: Add keywords to synergy.desktop
Author: "Joshua Honeycutt" <joshua.honeycutt@gmail.com>
Last-Update: 2016-12-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/res/synergy.desktop
+++ b/res/synergy.desktop
@@ -5,3 +5,4 @@
 Icon=/usr/share/icons/synergy.ico
 Type=Application
 Categories=Utility;
+Keywords=keyboard;mouse;sharing;network;share;
